config/config.example.toml

```diff
@@ -20,22 +20,40 @@ request_body_limit = 16_384
 # Main SQL data store credentials
 [master_database]
 username = "db_user"      # DB Username
-password = "db_pass"      # DB Password
+password = "db_pass"      # DB Password. Only applicable when KMS is disabled.
 host = "localhost"        # DB Host
 port = 5432               # DB Port
 dbname = "hyperswitch_db" # Name of Database
 pool_size = 5             # Number of connections to keep open
 connection_timeout = 10   # Timeout for database connection in seconds
+# Key ID for the KMS managed key used to decrypt the database password.
+# Only applicable when KMS is enabled.
+aws_key_id = ""
```

Comment:
move all `aws_key_id` and `aws_region` to single place under kms itself.

Reply:
I've opened #744 for this change. Marking this PR as blocked until it gets merged.